src/main/java/com/conveyal/r5/analyst/cluster/TravelTimeSurfaceTask.java

```diff
@@ -15,6 +14,9 @@
  * of travel time from the chosen origin to that destination.
  */
 public class TravelTimeSurfaceTask extends AnalysisTask {
+
+    // FIXME red flag - what is this enum enumerating Java types?
+
```

Comment:
Should we rename AnalysisTask.Type?

Reply:
It's not that we should rename the enum - it's that Java already has a type system, which allows checking if an object is an `instanceof` a class (which is also a bit of a hack) or even better, allows polymorphism. Different subclasses or implementations should just have methods with the same names that behave differently.

This is probably a remnant of using a single task-enqueue endpoint for both regional and single point tasks. I have created a separate issue for this:  #462